src/analysis/souffle/analyze_tag_checks.dl

```diff
@@ -0,0 +1,27 @@
+#ifndef SRC_ANALYSIS_SOUFFLE_ANALYZE_TAG_CHECKS_DL_
```

Comment:
This file is not used anywhere in this PR? If so, remove?

Reply:
Done.